src/app/jscaip/Minimax.ts

```diff
@@ -38,6 +38,12 @@ export abstract class PlayerMetricsMinimax<M extends Move,
                                            R extends Rules<M, S, L> = Rules<M, S, L>>
     extends Minimax<M, S, L, BoardValue, R>
 {
+
+  /**
+   * Assigns a metric for each player, as [player0, player1].
+   * A metric is just a number that the minimax will try to maximize.
+   * It can be positive, negative, or zero.
+   */
```

Comment:
Précise que ça doit être du même signe pour les deux, et que positif signifie avantageux (que ce soit pour joueur un ou deux).

Aussi, ferions nous un type de retour genre { playerZero: number, playerOne: number } ?

Reply:
Précision apportée !

le [A, A] pour des données des deux joueurs est assez standard, et notamment utilisé pour les scores, donc je serais pour garder cela ici, ça permet de retourner un .getScores(), puis ça permet si on le souhaite d'itérer sur les joueurs, là où un type spécifique requiert de réinventer un peu la roue si on veut le faire